src/main/kotlin/com/terraformation/backend/accelerator/model/ApplicationModel.kt

```diff
@@ -25,6 +30,38 @@ data class ExistingApplicationModel(
     val projectName: String,
     val status: ApplicationStatus,
 ) {
+  fun toGeoFeature(): SimpleFeature {
+    if (boundary == null) {
+      throw IllegalStateException("Application has no boundary")
+    }
+
+    val featureTypeBuilder = SimpleFeatureTypeBuilder()
+    featureTypeBuilder.setName("Accelerator Application")
+    featureTypeBuilder.add("the_geom", MultiPolygon::class.java)
+    featureTypeBuilder.add("applicationId", Long::class.java)
+    featureTypeBuilder.add("countryCode", String::class.java)
+    featureTypeBuilder.add("internalName", String::class.java)
+    featureTypeBuilder.add("organizationId", Long::class.java)
+    featureTypeBuilder.add("organizationName", String::class.java)
+    featureTypeBuilder.add("projectId", Long::class.java)
+    featureTypeBuilder.add("projectName", String::class.java)
+    featureTypeBuilder.add("status", String::class.java)
+
+    val featureType = featureTypeBuilder.buildFeatureType()
+
+    val featureBuilder = SimpleFeatureBuilder(featureType)
+    featureBuilder.set("the_geom", boundary.toMultiPolygon())
+    featureBuilder.set("applicationId", id.value)
+    countryCode?.let { featureBuilder.set("countryCode", it) }
+    featureBuilder.set("internalName", internalName)
```

Comment:
Won't this expose the internal name to end users?

Reply:
This is only an internal endpoint. But I think I probably should hide that behind a store and permission.

Reply:
Ah, I assumed it was meant to be end-user accessible since nothing was checking for permissions beyond the ability to read the application.